cli/src/commands/run_config_docker.py

```diff
@@ -12,35 +13,70 @@ def get_usage(self):
         return super(RunConfigDocker, self).get_usage() + " config"
 
     def get_options(self):
-        return [{"name": "config", "description": "path to the config to run"}]
+        return [{"name": "config",
+                 "description": "path to the config to run from the remote docker image"},
+                {"name": "from_local_code",
+                 "description": "use the local code to run the crawl",
+                 'optional': False}]
+
+    @staticmethod
+    def from_local_code_parse(args):
```

Comment:
Could you explain what is this method doing? I'm not sure I follow? It reads the second argument and cast it as a boolean?

Reply:
It jus a getter on the option. It translate a string into bool if needed